sda/processor.go

```diff
@@ -98,56 +93,49 @@ func (p *ServiceProcessor) Process(packet *network.Packet) {
 	log.Panic("Cannot handle message.")
 }
 
-// ProcessClientRequest takes a request from a client, calculates the reply
-// and sends it back.
-func (p *ServiceProcessor) ProcessClientRequest(si *network.ServerIdentity,
-	cr *ClientRequest) {
-	// unmarshal the inner message
-	mt, m, err := network.UnmarshalRegisteredType(cr.Data,
-		network.DefaultConstructors(network.Suite))
-	if err != nil {
-		log.Error("Err unmarshal client request:" + err.Error())
-		return
-	}
-	reply := p.GetReply(si, mt, m)
-	if err := p.SendRaw(si, reply); err != nil {
-		log.Error(err)
-	}
+// NewProtocol is a stub for services that don't want to intervene in the
+// protocol-handling.
+func (p *ServiceProcessor) NewProtocol(tn *TreeNodeInstance, conf *GenericConfig) (ProtocolInstance, error) {
+	return nil, nil
```

Comment:
OK - done. Later on I'd like to merge `RegisterProtocolSDA` and `RegisterProtocolService` and put a standard handler in the `ServiceProcessor.